packages/@sanity/dashboard/src/widget.css

```diff
@@ -21,7 +21,7 @@
 }
 
 .title {
-  composes: heading4 from "part:@sanity/base/theme/typography/headings-style";
+  composes: heading4 from 'part:@sanity/base/theme/typography/headings-style';
```

Comment:
Is there a reason for using single quotes?

Reply:
It’s prettier that’s doing it!

Reply:
Ah. You should disable prettier for CSS make it follow til stylelint rules. A lot of the changes on this PR was probably just prettier

Reply:
I’ve enabled that stylelint integration for Prettier in VSCode, though.

Reply:
![image](https://user-images.githubusercontent.com/406933/56799181-b4015200-6818-11e9-8781-38c1f180202b.png)

Reply:
Maybe we should put a line in `.editforconfig`

Reply:
I think the `stylelint-config-standard` which is used, prefers single quotes.